packages/common/src/chains/sepolia.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "sepolia",
+  "chainId": 11155111,
+  "networkId": 11155111,
+  "defaultHardfork": "istanbul",
+  "consensus": {
+    "type": "pow",
+    "algorithm": "ethash",
+    "ethash": {}
+  },
+  "comment": "PoW test network",
+  "url": "https://github.com/ethereum/ropsten",
+  "genesis": {
+    "hash": "0x25a5cc106eea7138acab33231d7160d69cb777ee0c2c553fcddf5138993e6dd9",
+    "timestamp": "0x6159af19",
+    "gasLimit": 30000000,
+    "difficulty": 131072,
+    "nonce": "0x000000000000000",
+    "extraData": "0x5365706f6c69612c20417468656e732c204174746963612c2047726565636521",
+    "stateRoot": "0x5eb6e371a698b8d68f665192350ffcecbbbf322916f4b51bd79bb6887da3f494"
```

Comment:
Did you took over `hash` and `stateRoot` here from somewhere or did you calculate yourself?

Can you confirm the correctness of both?

Reply:
I'll need to compare in more detail but here's what Besu is using for Genesis config.https://github.com/hyperledger/besu/blob/main/config/src/main/resources/sepolia.json

Reply:
I got them from this comment on the original geth PR - https://github.com/ethereum/go-ethereum/pull/23730#issuecomment-942646618. I'll see if I can confirm them with our client

Reply:
Ah, ok, but that's also already a solid source. 👍